airflow/providers/databricks/operators/databricks_repos.py

```diff
@@ -90,7 +90,7 @@ def __init__(
             self.git_provider = self.__detect_repo_provider__(git_url)
             if self.git_provider is None:
                 raise AirflowException(
-                    "git_provider isn't specified and couldn't be guessed for URL {git_url}"
+                    f"git_provider isn't specified and couldn't be guessed for URL {git_url}"
```

Comment:
@alexott I took care of fixing this since it was missed in the initial cleanup.

Reply:
Thank you